docker/ceph/start-ceph.sh

```diff
@@ -38,6 +38,13 @@ echo 'vstart.sh completed!'
 # Configure Object Gateway:
 /docker/set-rgw.sh
 
+# Create CephFS filesystem
+if [[ "$MDS" -gt 0 ]]; then
+  ceph osd pool create cephfs_data 8
```

Comment:
Please prepend ceph command with ```"$CEPH_BIN"/```:
```
"$CEPH_BIN"/ceph ...
```
so it can work with built-from-scratch ceph also.

Reply:
thanks!